compiler/Makefile

```diff
@@ -25,12 +26,15 @@ parser.ml parser.mli: parser.mly
 
 .PHONY: clean
 clean:
-	rm -f scanner parser scanner.ml parser.ml parser.mli *.cmo *.cmi
+	rm -f odds generator scanner parser scanner.ml parser.ml parser.mli *.cmo *.cmi
 
 # Generated by ocamldep *.ml *.mli
+generator.cmo: scanner.cmo parser.cmi ast.cmi
+generator.cmx: scanner.cmx parser.cmx ast.cmi
 parser.cmo: ast.cmi parser.cmi
 parser.cmx: ast.cmi parser.cmi
-parser.cmi: ast.cmi
 scanner.cmo: parser.cmi
 scanner.cmx: parser.cmx
-ast.cmi:
+ast.cmi: 
+parser.cmi: ast.cmi
+sast.cmi: ast.cmi
```

Comment:
included in other PR but let's remove SAST from this branch if not implemented yet